Makefile

```diff
@@ -181,6 +181,8 @@ coverage: $(TESTS_TARGET) $(COMPILE_CLI_TARGET)
 	mv default.profraw tests.profraw
 	$(TESTS_TARGET) --fflags=true
 	mv default.profraw tests-flags.profraw
+	$(TESTS_TARGET) --fflags=true,DebugLuauDeferredConstraintResolution=true || true
```

Comment:
Does this warrant a comment? Can we write it in such a way this fails if the program returns `0` forcing us to clean up this hack?

Reply:
Good idea. Fixed.